core/src/main/java/com/sequenceiq/cloudbreak/controller/ImageCatalogV3Controller.java

```diff
@@ -51,9 +51,13 @@
     }
```

Comment:
This concat logic should be moved to ImageCatalogService, otherwise the default images won't be returned in the v1 endpoint.